### PR TITLE
docs: improve README examples for `stats/base/dists/t` namespace

### LINKED ISSUE #1645: [RFC]: improve README examples of `stats/base/dists/t` namespace
### Description

This RFC proposes to improve the [README examples](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/stats/base/dists/t#examples) of the `stats/base/dists/t` namespace package. Currently, namespace package READMEs only include minimal examples which are not particularly informative and do not showcase namespace functionality.

Historically, namespace examples were not given much effort due to the rapidly changing nature of the project; however, now that development and organization has settled, it would be good to revisit these packages and add proper examples to better communicate to users how to use namespace contents.

For those wanting to contribute to this issue, you can use the following steps:

- Browse through the [contents](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/stats/base/dists/t) of the `stats/base/dists/t` namespace and identify interesting functions and packages to showcase.
- Update the example code block in the "Examples" section of the README for the `stats/base/dists/t` namespace and open a PR.
- When updating the README, examples should follow existing project conventions (e.g., ES5).
- Examples may `require` functionality from other packages to help make examples more interesting.

### Related Issues

None.

### Questions

No.

### Other

No.

### Checklist

- [X] I have read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [X] Searched for existing issues and pull requests.
- [X] The issue name begins with `RFC:`.

### PR DESCRIPTION
**Pull request by Jai0401**
Resolves #1645 

## Description

> What is the purpose of this pull request?

This pull request:
Improve README examples of `stats/base/dists/t` namespace

## Related Issues

> Does this pull request have any related issues?
No.

This pull request:

-   fixes #1645 

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/stats/base/dists/t/README.md [hunk 1]
```diff
@@ -112,6 +112,23 @@ var objectKeys = require( '@stdlib/utils/keys' );
 var t = require( '@stdlib/stats/base/dists/t' );
 
 console.log( objectKeys( t ) );
+
```
Comment (kgryte) [inline lib/node_modules/@stdlib/stats/base/dists/t/README.md]:
```suggestion
```

File: lib/node_modules/@stdlib/stats/base/dists/t/examples/index.js [hunk 2]
```diff
@@ -22,3 +22,20 @@ var objectKeys = require( '@stdlib/utils/keys' );
 var t = require( './../lib' );
 
 console.log( objectKeys( t ) );
+
```
Comment (kgryte) [inline lib/node_modules/@stdlib/stats/base/dists/t/examples/index.js]:
```suggestion
```

File: lib/node_modules/@stdlib/stats/base/dists/t/README.md (hunk 1 above)
Comment (kgryte) [reply lib/node_modules/@stdlib/stats/base/dists/t/README.md]: You should remove the `utils/keys` import.

File: lib/node_modules/@stdlib/stats/base/dists/t/README.md [hunk 3]
```diff
@@ -112,6 +112,23 @@ var objectKeys = require( '@stdlib/utils/keys' );
...
+console.log( 'PDF at x = ' + xValue + ': ' + pdfResult );
+
+// Evaluate the cumulative distribution function (CDF) at a specific value
+var cdfResult = t.cdf( xValue, degreesOfFreedom );
+console.log( 'CDF at x = ' + xValue + ': ' + cdfResult );
+
+// Get the mean and variance of the t distribution
+var meanValue = t.mean( degreesOfFreedom );
+var varianceValue = t.variance( degreesOfFreedom );
+console.log( 'Mean: ' + meanValue + ', Variance: ' + varianceValue );
```
Comment (kgryte) [inline lib/node_modules/@stdlib/stats/base/dists/t/README.md]:
```suggestion
var dof = 3;
var x = 1.5;

// Evaluate the probability density function (PDF) at a specific value:
var res = t.pdf( x, dof );
console.log( 'PDF at x = ' + x + ': ' + res );

// Evaluate the cumulative distribution function (CDF) at a specific value:
res = t.cdf( x, dof );
console.log( 'CDF at x = ' + x + ': ' + res );

// Get the mean and variance of the t distribution:
var mu = t.mean( dof );
var v = t.variance( dof );
console.log( 'Mean: ' + mu + ', Variance: ' + v );
```

Follow project conventions in preferring shorter variable names.

**Comment by Jai0401** [comment]
@kgryte Thank you for your feedback! I have addressed all the issues you mentioned in your review. Please take a look at the changes, and let me know if everything looks good now. If you have any further suggestions or concerns, feel free to share them. Thanks again!

**Review by Planeshifter — APPROVED** [review]
Looks alright; let's land this change. Thanks!

**Pull request merged**